app/pipedrive/api.py

```diff
@@ -21,10 +23,10 @@
 session = requests.Session()
 
 
-async def pipedrive_request(url: str, *, method: str = 'GET', data: dict = None) -> dict:
-    r = session.request(
-        method=method, url=f'{settings.pd_base_url}/api/v1/{url}?api_token={settings.pd_api_key}', data=data
-    )
+async def pipedrive_request(url: str, *, method: str = 'GET', args: dict = None, data: dict = None) -> dict:
```

Comment:
args are `[foo, bar]`. kwargs are `{'foo': 'foo', 'bar': 'bar'}`

also call it `query_kwargs` or something sensible